### ISSUE TITLE
[Accessibility]: Show time in line chart

### ISSUE THREAD

**Issue by hrithik098** [carbon-design-system/carbon-charts#1659]
### Name

Hrithik Yadav

### Are you an IBM employee?

- [ ] Yes

### Environment

Browser

### What happened? What did you expect to happen instead?

The issue I'm facing is that when using Carbon Charts for line series visualizations, the time component is not displayed in the visualizations. Specifically, when visualizing time series data, only the date is shown on the x-axis, and I expected both date and time to be displayed to provide more granular information. If not on the x-axis, at least in tool tip over the point or in tabular form I should be able to see time also.

### What WCAG 2.1 checkpoint does the issue violate?

Iit doesn't directly violate a specific WCAG 2.1 checkpoint, it could affect users who rely on precise time-related data.

### Version

@carbon/charts: "^1.7.6

### Data & options used

```js
options={{
    title: props.title,
    axes: {
      bottom: {
        title: props.xLabel,
        mapsTo: "time",
        scaleType: ScaleTypes.TIME,
      },
      left: {
        mapsTo: "value",
        title: props.yLabel,
        scaleType: ScaleTypes.LINEAR,
      },
    },
    timeScale: {
      showDayName: true,
    },
    data: {
      loading: loading,
    },
    curve: "curveMonotoneX",
    height: "600px",
  }}

data=[
    {
        "value": 81.95625305,
        "time": "2023-09-01T23:59:59"
    },
    {
        "value": 83.43281555,
        "time": "2023-09-02T23:59:59"
    },
    {
        "value": 84.56311798,
        "time": "2023-09-03T23:59:59"
    },
    {
        "value": 85.70030975,
        "time": "2023-09-04T23:59:59"
    },
    {
        "value": 86.44625092,
        "time": "2023-09-05T23:59:59"
    },
    {
        "value": 88.15436554,
        "time": "2023-09-06T23:59:59"
    },
    {
        "value": 89.12812042,
        "time": "2023-09-07T23:59:59"
    },
    {
        "value": 90.76218414,
        "time": "2023-09-08T23:59:59"
    },
    {
        "value": 91.88811493,
        "time": "2023-09-09T23:59:59"
    },
    {
        "value": 93.44281006,
        "time": "2023-09-10T23:59:59"
    },
    {
        "value": 94.32093811,
        "time": "2023-09-11T23:59:59"
    },
    {
        "value": 94.45030975,
        "time": "2023-09-12T23:59:59"
    },
    {
        "value": 94.46938324,
        "time": "2023-09-13T23:59:59"
    }
]
```

### What priority level would this issue be in your opinion?

P3
